lib/bots/versionbot.ts

```diff
@@ -212,11 +212,38 @@ export class VersionBot extends GithubBot.GithubBot {
             // Go through all the commits. We're looking for, at a minimum, a 'change-type:' tag.
             for (let commit of commits) {
                 const commitMessage: string = commit.commit.message;
-                const invalidCommit = !commitMessage.match(/^change-type:\s*(patch|minor|major)\s*$/mi);
 
-                if (!invalidCommit) {
-                    changetypeFound = true;
-                    break;
+                // Split the commits up into lines, and find the last line with any whitespace in.
+                // Whilst we tend to ask for:
+                //  <header>
+                //
+                //  <body>
+                //
+                //  <footer>
+                // This code will actually let you get away with:
+                //  <header>
+                //
+                //  <footer>
+                // As sometimes a development patch may be self-explanatory in the header alone.
+                const lines = commitMessage.split('\n');
+                let index = 0;
+                for (index = lines.length - 1; index >= 0; index -= 1) {
+                    if (lines[index].match(/$\s*^/)) {
```

Comment:
A good catch.. yet it's worked with all the tests I've thrown at it.
Yes, unit and integration tests are on the list, but it was deemed more important to get them going first. Lots of things on the list...